signal.py

```diff
@@ -318,13 +318,26 @@ def shutdown():
 
 def smsg_cmd_cb(data, buffer, args):
     if len(args) == 0:
-        prnt("Not enough arguments! Try /help smg")
-    elif " " not in args:
-        get_buffer(args, False)
+        prnt("Usage: /smsg [number | group | *]")
     else:
-        number, message = args.split(" ", 1)
-        # getSignal().sendMessage(message, dbus.Array(signature="s"), number)
-        show_msg(number, None, message, False)
+        try:
+            if args in contacts:
+                identifier = args
+                group = None
+            elif args == '*':
+                prnt('List of all available Signal groups:')
+                for group in groups:
+                    prnt(groups[group]['name'])
+                identifier = False
+            else:
+                for group in groups:
+                    if groups[group]['name'] == args:
+                        identifier = group
+            if identifier:
+                buf = get_buffer(identifier, group is not None)
+        except UnboundLocalError:
```

Comment:
This feels like kind of a hack; can we just assign the contact or group to be None and test for that?